src/types/Transaction.tsx

```diff
@@ -0,0 +1,48 @@
+import { Category } from './Category';
+import { PaymentFrequency } from './PaymentFrequency';
+
+enum AttachmentType {
+  Camera = 'camera',
+  Image = 'image',
+  Document = 'document',
+}
+
+enum AccountType {
+  Personal = 'personal',
+  Business = 'business',
+  Joint = 'joint',
+  Corporate = 'corporate',
+}
+
+enum Currency {
+  USD = 'USD',
+  IDR = 'IDR',
+  JPY = 'JPY',
+  RUB = 'RUB',
+  EUR = 'EUR',
+  WON = 'WON',
+}
+
+export interface Transaction {
+  category: Category;
+  description?: string;
+  attachments?: Array<{
```

Comment:
It's better to type attachments like this: `attachments: Attachment[];`
where 

```
interface Attachment {
  type: AttachmentType;
  url: string;
}
```

Reply:
Fixed